tests/sgx_zerobase/enc/create_zerobase.conf

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Enclave settings:
+Debug=1
```

Comment:
Shouldn't the conf file specifies the 0-base options?

Reply:
Modified the test cases to address these scenarios.